admin/test/test_acceptance.py

```diff
@@ -205,3 +207,19 @@ def test_non_json_message(self):
             )],
             self._convert(NON_JSON_JOURNAL_EXPORT),
         )
+
+
+class CommonOptionsParser(TestCase):
+    """
+    Tests for ``CommonOptions``
+    """
+    def test_invalid_version(self):
+
+        arg_options = "--distribution ubuntu-14.04 --provider AWS " \
+            "--config-file acceptance.yml " \
+            "flocker-version invalid-version"
+        common_options = CommonOptions(arg_options)
+        self.assertRaises(
+            UsageError,
+            common_options.parseOptions, arg_options
+        )
```

Comment:
And if you drop to pdb here you'll see that the exception is different than expected because you need to supply a `list` rather than a `str` to parseOptions.

```
(Pdb) exception
UsageError('Wrong number of arguments.',)
(Pdb) list
220  	        common_options = CommonOptions(arg_options)
221  	        exception = self.assertRaises(
222  	            UsageError,
223  	            common_options.parseOptions, arg_options
224  	        )
225  ->	        import pdb; pdb.set_trace()
```

``assertRaises`` returns the exception so that you can make further assertions about the attributes of the exception.